Code/GraphMol/Deprotect/Deprotect.h

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (C) 2020 Brian P Kelley, Joann Prescott-Roy
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#ifndef RDK_DEPROTECT_LIBRARY
+#define RDK_DEPROTECT_LIBRARY
+
+#include <RDGeneral/export.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/ChemReactions/Reaction.h>
+#include <string>
+
+namespace RDKit
+{
+
+/*! Data for Deprotecting molecules
```

Comment:
Maybe this code should all go in an RDKit::Deprotect namespace?
We've been putting new libraries into their own namespaces and it seems to make sense.